storage/books.json

```diff
@@ -0,0 +1 @@
+["{\"title\":\"sd\",\"author\":\"sd\"}","{\"title\":\"hola\",\"author\":\"hola\"}","{\"title\":\"h\",\"author\":\"h\"}","{\"title\":\"rails\",\"author\":\"backend\"}","{\"title\":\"rails\",\"author\":\"backend\"}","{\"title\":\"rails\",\"author\":\"backend\"}","{\"title\":\"Catcher In The Rye\",\"author\":\"Salinger\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"infidelity\",\"author\":\"eugene\"}","{\"title\":\"mastery\",\"author\":\"robert\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"Atomic Habbits\",\"author\":\"Eugene\"}","{\"title\":\"Mastery\",\"author\":\"Robert\"}","{\"title\":\"Mastery\",\"author\":\"Robert\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"z\",\"author\":\"z\"}","{\"title\":\"z\",\"author\":\"z\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"z\",\"author\":\"z\"}","{\"title\":\"x\",\"author\":\"x\"}","{\"title\":\"z\",\"author\":\"z\"}","{\"title\":\"x\",\"author\":\"x\"}","{\"title\":\"c\",\"author\":\"c\"}","{\"title\":\"v\",\"author\":\"v\"}","{\"title\":\"z\",\"author\":\"z\"}","{\"title\":\"x\",\"author\":\"x\"}","{\"title\":\"z\",\"author\":\"z\"}","{\"title\":\"x\",\"author\":\"x\"}","{\"title\":\"x\",\"author\":\"x\"}","{\"title\":\"a\",\"author\":\"a\"}","{\"title\":\"ty\",\"author\":\"ty\"}","{\"title\":\"d\",\"author\":\"d\"}"]
```

Comment:
- [ ] Please remove the `redundancy` from your code, I tried to run the application and went with option 1 to `list` down all the books, but I was not able to get the book information that you have stored, instead I was getting the `blank` author, title data. 

![Screenshot from 2023-11-01 02-52-06](https://github.com/Omwomo/school-library/assets/66476329/f657370b-d06d-4d78-b0ba-f790aa423400)

- [ ] The second problem, that I noted down is that you haven't added the `try` catch in your code to handle the exceptions and errors in your code, just imagine, by mistake user deleted the `books.json` file from your folder and if user run the application, it will give an error regarding, file not found. I would like to suggest you find a way, if the file doesn't exist then on the spot the file is created by your application and serves its purpose.

- [ ] The Third problem that I faced was, if a person manually adds the empty array inside this file `so` that the application should work smoothly, I noted that your data is not converted to `JSON` format properly. Because the data you are preserving is not in the correct format. maybe this is the reason, while listing the book, I was not able to get the preserve data on the console display. Kindly fix the things test your application thorougly with all edge cases.

![Screenshot from 2023-11-01 02-56-08](https://github.com/Omwomo/school-library/assets/66476329/37b9479e-5060-4c78-8a2e-7dd87869e22d)